lib/markdown.js

```diff
@@ -1,15 +1,21 @@
-const marked = require('marked');
+const MarkdownIt = require('markdown-it');
 
 module.exports = class Markdown {
     constructor(markdown) {
         this.markdown = markdown;
     }
 
     toString() {
-        return this.markdown;
+        return this.markdown.trim();
     }
 
     toHtml() {
-        return marked(this.markdown);
+        const markDownIt = new MarkdownIt();
+
+        return markDownIt.render(this.markdown.trim());
```

Comment:
```suggestion
        return markDownIt.render(this.toString());
```

Reply:
I would not do that change because it add a coupling of code that is not necessary. If I, for some reason, change the code of `toString()` I have to pay attention that I also refactor the `toHtml()` method. They both currently use `this.markdown.trim()` but this is just accidentally.